lib/server/sthServer.js

```diff
@@ -59,6 +61,7 @@ function doStartServer(host, port, callback) {
 
     server.on('request-internal', function(request, event, tags) {
         if (tags.error) {
+        	processedRequestsWithError++;
```

Comment:
Bad indentation?

Reply:
Yes. It seems so.

Reply:
Corrected the indentation. Fixed in 5fe4c92